src/java.base/share/classes/com/sun/crypto/provider/DHPublicKey.java

```diff
@@ -48,28 +48,28 @@ final class DHPublicKey implements PublicKey,
 javax.crypto.interfaces.DHPublicKey, Serializable {
 
     @java.io.Serial
-    static final long serialVersionUID = 7647557958927458271L;
+    private static final long serialVersionUID = 7647557958927458271L;
 
     // the public key
-    private BigInteger y;
+    private final BigInteger y;
 
     // the key bytes, without the algorithm information
-    private byte[] key;
+    private final byte[] key;
```

Comment:
Same comment about re-assignment, please don't add the final keyword to this field.